.github/workflows/build_msi_installer.yml

```diff
@@ -108,6 +108,19 @@ jobs:
           setupType: promptflow_with_extra
           scriptPath: ${{ env.testWorkingDirectory }}
 
+      - name: Install wheel
```

Comment:
should we delete this: **name: Setup and Install dev promptflow**

and install promptflow-tools in **- name: Install wheel**